wordpress-mobile/WordPress-Android
# Issue 15910 - Fix notification comment details content not updated after editing

Fixes #15910

Edited notification comment does not show the updated content in the details screen.

<p float="left">
<img width="250" alt="Screen Shot 2022-02-09 at 4 20 25 AM" src="https://user-images.githubusercontent.com/14964993/153142506-13b73257-5a75-4631-8951-3556567b351e.png">
<img width="250" alt="Screen Shot 2022-02-09 at 4 20 31 AM" src="https://user-images.githubusercontent.com/14964993/153142517-eec15318-ccac-4347-ac9d-f282ffa91c1b.png">
<img width="250" alt="Screen Shot 2022-02-09 at 4 20 37 AM" src="https://user-images.githubusercontent.com/14964993/153142597-1423ae8d-3979-40f7-9caa-f42f03cea622.png">
</p>

To test:
1 - Login with an user that has at least one notification of a comment;
2 - Select any site;
3 - Tap on "Notifications" bottom navigation icon to open Notifications screen;
4 - Tap on "Comments" tab inside Notifications screen;
5 - Tap on the comment you want to test;
6 - Tap on "More" button;
7 - Tap on "Edit" item. The notification comment edit should be opened in the new comment editor;
8 - Edit the comment text;
9 - Tap on "Done";
10 - The comment text should be updated in the comment details screen.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
\-

3. What automated tests I added (or what prevented me from doing so)
Current organization of the changed classes prevents us from adding unit tests right now. Refactoring them in the future will allow us to do so.

PR submission checklist:
